docs/user_guide.md

```diff
@@ -63,35 +61,33 @@ cd noresm-land-sites-platform
+In the graphical user interface (= GUI) at [localhost:8080](http://localhost:8080) you can easily set up and run model experiments. This interface has access to the Docker containers you started in step 1, and communicates with the models via the [API](https://noresmhub.github.io/noresm-land-sites-platform/#api). 
+
+In the GUI, you will see a list and map of integrated sites, as well as a list of your cases (model experiments) once you create some. You can also [run a simulation for a new site with your own input data](https://noresmhub.github.io/noresm-land-sites-platform/land-sites/). If you are doing a quick test with default settings, just pick a site and click `create case`, `submit` and then `run`. A default test case of 1 year should take ~10-20 mins. The GUI should look like this once you have chosen a site and have submitted and started to run a case:
 
 ![GUI screenshot](img/LTK-ALP1-screenshot.png)
 
 *Figure 1: Screenshot from the GUI with a case created for the ALP1 site. Edited for readability.*
 
 #### 2.1 Choose a [site](https://noresmhub.github.io/noresm-land-sites-platform/land-sites/) 
 
-...by clicking either a button or a point on the map! 
-
-If you need a different site, you can [request one via GitHub](https://github.com/NorESMhub/noresm-land-sites-platform/issues/new?assignees=&labels=enhancement&template=new-site.md&title=New+site+request%3A). This requires the developers to do some manual work, so you should have a clear reason to request a new site.
+...by clicking either a button or a point on the map! Or [add a new site with your own data](https://noresmhub.github.io/noresm-land-sites-platform/land-sites/).
 
 #### 2.2 Download site data button (optional)
 
-The input data the models need is already available in the container, but with the `download site data` button you can download it to somewhere else if you wish. We provide notebooks under `/notebooks/plot_input_data/` that you can open in Jupyterlab on [localhost:8888](localhost:8888) to familiarize yourself with some of the data that goes into a simulation, or as inspiration for making your own plots if you provided your own, improved input data. This is a good thing to do while your case is running.
+For the integrated sites, the input data the models need is already available in the container. With the `download site data` button you can download it to somewhere else if you wish. We provide notebooks under `/notebooks/plot_input_data/` that you can open in Jupyterlab on [localhost:8888](localhost:8888) to familiarize yourself with some of the data that goes into a simulation, or as inspiration for making your own plots if you provided your own, improved input data. This is a good thing to do while your case is running.
```

Comment:
Paths like `notebooks/...` must be relative to the repo root, so the first slash can go.